crates/bevy_math/Cargo.toml

```diff
@@ -32,6 +34,8 @@ glam = { version = "0.27", features = ["approx"] }
 
 [features]
 default = ["rand"]
+# Enable reflection for math types.
+reflect = ["dep:bevy_reflect"]
```

Comment:
Note that packages like `bevy_ECS` don't specify a `reflect` feature. I believe they just mark the dependency as optional. We might want to do the same here

Reply:
That's probably clearer and easier.